bundles/org.eclipse.ui.ide.application/src/org/eclipse/ui/internal/ide/application/IDEApplication.java

```diff
@@ -24,9 +24,11 @@
 import java.io.FileInputStream;
 import java.io.FileOutputStream;
 import java.io.IOException;
+import java.io.InputStream;
 import java.io.OutputStream;
 import java.net.InetAddress;
 import java.net.MalformedURLException;
+import java.net.URISyntaxException;
 import java.net.URL;
 import java.nio.file.Files;
 import java.nio.file.Path;
```

Comment:
nio was introduced by https://github.com/eclipse-platform/eclipse.platform.ui/pull/2347/files#diff-6b2613840cdec17a5b7ab1f515650cc101c4637884833261d1353c6e0b9d7535R32.
I think completely getting rid of it safe.